addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -194,6 +193,10 @@ def _discuss_channel_member_format(self, fields=None):
                 data['custom_notifications'] = member.custom_notifications
             if 'mute_until_dt' in fields:
                 data['mute_until_dt'] = member.mute_until_dt
+            if 'fetched_message_id' in fields:
+                data['fetched_message_id'] = {'id': member.fetched_message_id.id} if member.fetched_message_id else False
+            if 'seen_message_id' in fields:
+                data['seen_message_id'] = {'id': member.seen_message_id.id} if member.seen_message_id else False
```

Comment:
Look a bit strange to have a dict with only an id inside a  `_id` field. Those dict could be skipped.

Reply:
Or we just add the guideline to name relations "_id" and "_ids" in JS just like we do in python so we instantly fix all consistency issues.